### ISSUE TITLE
Remove dislike counter and ratio bar

### ISSUE THREAD

**Issue by TheCultLeader666** [FreeTubeApp/FreeTube#1904]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
With the removal of the dislike count from YouTube, the UI for counting dislikes and the ratio bar that shows the difference in likes vs. dislikes has stopped working
**Describe the solution you'd like to see implemented**
Remove the dislike button and bar
**Describe alternatives you've considered**
YouTube may be pressured to bring back dislike count, in which case this will stop being applicable.
**Screenshots**
![image](https://user-images.githubusercontent.com/65368815/142492091-900850a4-e651-4ad1-a16a-f2b684839d0e.png)

**Comment by peepopoggers** [comment]
dislikes are still available, so not yet.

**Comment by TheCultLeader666** [comment]
> dislikes are still available, so not yet.

If you're going to keep the bar, you should at least make use of it. It's currently just an eyesore with no function.

**Comment by peepopoggers** [comment]
I just explained to you, dislikes are still available for some videos.

**Comment by jpsnyder** [comment]
Perhaps once the dislike count is no longer accessible from the API, the dislike count could be estimated just like https://github.com/Anarios/return-youtube-dislike is planning to do?

**Comment by TheCultLeader666** [comment]
> I just explained to you, dislikes are still available for some videos.

Haven't seen them on a single one since the change

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I did

**Comment by Elendil211** [comment]
Another option would be to display the like/view ratio, or the like/comment ratio, maybe with a modifier (*10000 or something like that).

**Comment by peepopoggers** [comment]
That is not really effective because some videos have more comments than others and it is unrelated to how good or bad the video was. For example I think there would be a bigger ratio (ratio) on a big news story than a music video, however the music video may be more disliked. Inaccuracies like this make it unreliable. Another option is to do what the 'bring back likes' (or whatever it's called) extension is doing. I am not sure how that extension works, but if they have an API then it can be implemented.

**Comment by lazytownfan** [comment]
I think we need some time to tell if dislikes are going to be permanently removed from YouTube and totally inaccessible via any API.

For put things in perspective on how the dislike progress is like, it is very uneven.  I will sometimes see both the like and dislike count YouTube proper (i.e., visiting `youtube.com` in the browser).  However, there is seemingly no rhyme or reason why this happens, as there are other times where the dislikes are gone and I only see the like count.

On Invidious (at least on the `yewtu.be` instance), I have been able to see both the like and dislike count - except just now when I tried to test this again - both likes and dislikes are set to 0.  As with YouTube proper, I am not sure why this is still happening (or just happened?).

In FreeTube, I can sometimes "cheat" my way into seeing the dislikes.  I prefer to do this by opening up the target video in a new window ... by continuously pressing <kbd>Ctrl</kbd>+<kbd>R</kbd> on the keyboard.  There are times where the dislikes are shown immediately, others where the dislikes appear after doing this 2-3 times, and I rarely have to do this more than 6-8 times in a row.

I'm using FreeTube via the `freetube` source package on the AUR on Manjaro (Unstable branch).
